### ISSUE TITLE
How to install this plugin on jenkins, i cant find in Available plugin list

### ISSUE THREAD

**Issue by kankhand** [jenkinsci/tikal-multijob-plugin#173]
i am using jenkins version. 2.154, Cant find way to install this plugin. Please advise.

**Comment by yorammi** [comment]
Are you sure it is not already installed and just disabled?

**Comment by yorammi** [comment]
I'm not using jenkins/jenkins image at all. I prefer working with https://github.com/odavid/my-bloody-jenkins which is much better as a start-point for establishing a new Jenkins server.

**Comment by yorammi** [comment]
And if you're really into testing new solutions, try Tikal's new project (led by me): [TCI](https://github.com/TikalCI/tci-server). With it, you can establish a configurable (Jenkins-as-code) Jenkins server withing few minutes. Please notice that it is still in pre-release version and not well documented yet.
It also contains pipeline-step called **parallelPhase**, which is a possible replacement for **tikal-multiJob-plugin**, in its **tci-library** shared Jenkins-Pipeline-library.

**Comment by kankhand** [comment]
Can you share the hpi file for this plugin?

**Comment by yorammi** [comment]
tci-library is not a plugin, it is a Pipeline shared library.
If you refer to tikal-multiJob-plugin, you can clone/fork the https://github.com/jenkinsci/tikal-multijob-plugin repository

**Comment by kankhand** [comment]
I can see only 2 branches, master and improvements.
and how to use this library in jenkins.
your help is much appreciated.

Actually I want to create MultiJob Project for my pipeline jobs but in current multijob plugin of jenkins, We are not able to run pipeline jobs in that plugin.

**Comment by kankhand** [comment]
and i am talking about  tikal-multiJob-plugin..Do we have hpi file for that plugin.

**Comment by yorammi** [comment]
there are only those branches indeed.
The latest version of the plugin has a significant change, which is that it finally can run pipeline jobs!

**Comment by kankhand** [comment]
yes, So, Can you get me hpi file of that plugin (read tikal-multiJob-plugin)???
